ReactAndroid/build.gradle

```diff
@@ -355,6 +352,9 @@ dependencies {
     api("com.squareup.okhttp3:okhttp:${OKHTTP_VERSION}")
     api("com.squareup.okhttp3:okhttp-urlconnection:${OKHTTP_VERSION}")
     api("com.squareup.okio:okio:1.15.0")
+    if(JSEngineUsed=='JSC'){
+    compile 'org.webkit:android-jsc:r174650'
```

Comment:
indent it with a tab

Reply:
follow file formatting

---
In reply to: [295770583](https://github.com/microsoft/react-native/pull/93#discussion_r295770583) [](ancestors = 295770583)